design-tokens/tokens/layer3/base.json

```diff
@@ -4,7 +4,7 @@
       "overlays": {
         "maxWidth": {
           "default": {
-            "value": "{echoes.dimension.size.5000}",
+            "value": "{echoes.dimension.width.5000}",
```

Comment:
I thought you wanted to add a layer 3 design token for the button heights

Reply:
Fixed